controllers_v2/invoice.ctrl.go

```diff
@@ -174,6 +175,7 @@ func (controller *InvoiceController) AddInvoice(c echo.Context) error {
 		PaymentHash:    invoice.RHash,
 		PaymentRequest: invoice.PaymentRequest,
 		ExpiresAt:      invoice.ExpiresAt.Time,
+		CreatedAt:      invoice.CreatedAt,
```

Comment:
why is it `CreatedAt` here but `ExpiresAt.Time` above?

Reply:
expiresAt is nullable, you can create an invoice that does not expire